### PR TITLE
apache-arrow 3.0.0

### PR DESCRIPTION
**Pull request by nealrichardson**
See also https://github.com/r-windows/rtools-packages/pull/184

### CODE REVIEW

**Comment by jeroen** [comment]
Thanks this looks good

**Comment by jeroen** [comment]
Do you really intend to enable both mimalloc and jemalloc?

**Comment by nealrichardson** [comment]
Yes, in the upcoming release you can configure which allocator you want to use

File: Formula/apache-arrow-static.rb [hunk 1]
```diff
@@ -42,6 +37,7 @@ def install
       -DARROW_PARQUET=ON
       -DARROW_BUILD_SHARED=OFF
       -DARROW_JEMALLOC=ON
+      -DARROW_MIMALLOC=ON
```
Comment (nealrichardson) [inline Formula/apache-arrow-static.rb:40]: FWIW I found that mimalloc didn't compile on my M1 mac, not sure if you've seen similar. So we may need to make this conditional.

Comment (jeroen) [reply Formula/apache-arrow-static.rb:40]: Havent' tried the R package, but building arrow seems to work.

**Comment by jeroen** [comment]
OK you were right, I was able to build libarrow with `-DARROW_MIMALLOC=ON` however the R package would immediately crash when loaded. I have now updated the formula to not enable mimalloc on arm.

https://github.com/autobrew/homebrew-cran/blob/fea37576b5815315a64f3cc9561f2db6bfc2e3c2/Formula/apache-arrow-static.rb#L55

**Pull request closed without merging**